static/components/TaskDetails.jsx

```diff
@@ -8,21 +8,21 @@ import PollingMixin from "../mixins/polling";
 import TaskSummary from "./TaskSummary";
 import TimeSince from "./TimeSince";
 import { browserHistory } from 'react-router';
+import PropTypes from 'prop-types';
 
 var moment = require('moment');
 
 var Progress = React.createClass({
-  propTypes: {
-    value: React.PropTypes.number.isRequired,
-  },
-
   render() {
     return (
       <span className="progress" style={{width: this.props.value + '%'}} />
     );
   }
 });
 
+Progress.propTypes = {
```

Comment:
Is this change necessary? It seems like `propTypes` should be an object (as it is above) when using `React.createClass`, while it's used as a property (the change you've made here) on the class when declaring a component with `extends React.Component`